main/get.c

```diff
@@ -829,7 +830,8 @@ extern int cppGetc (void)
 				int next = getcFromInputFile ();
 
 				if (next == NEWLINE)
-					continue;
+					// continue; <-- this may exit returning the backslash, which si wrong.
```

Comment:
It seems that this change is more generic fix than "Extract return types of functions. Correct all unit tests. ".
If possible I would like to you to make a separate commit only for this change.
I would like to see a test case specialized to this change.
Here this change means "changes for main/get.c".

main/get.c is used not only C/C++ but also other parsers.